<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>571</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>551</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>70</y>
      <width>191</width>
      <height>147</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Filas: </string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="Button_agregar_fila">
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_reducir_fila">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Columnas: </string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="button_agregar_column">
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_reducir_column">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="button_imprimir">
         <property name="text">
          <string>imprimir</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Button_print_combinatoria">
         <property name="text">
          <string>Combinar</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="button_limpiar">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>290</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Limpiar lista</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_2">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>250</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>250</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Cantidad de etiquetas: </string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>350</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Guardar en</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivos">
    <property name="title">
     <string>Archivos</string>
    </property>
    <addaction name="actionNuevo"/>
    <addaction name="actionAbrir"/>
    <addaction name="actionGuardar"/>
    <addaction name="actionGuardar_como"/>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuAcerca">
    <property name="title">
     <string>Acerca</string>
    </property>
    <addaction name="actionAcerca"/>
   </widget>
   <addaction name="menuArchivos"/>
   <addaction name="menuAcerca"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAcerca">
   <property name="text">
    <string>Acerca</string>
   </property>
  </action>
  <action name="actionNuevo">
   <property name="text">
    <string>Nuevo</string>
   </property>
  </action>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar</string>
   </property>
  </action>
  <action name="actionGuardar_como">
   <property name="text">
    <string>Guardar como</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
